CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java

```diff
@@ -699,7 +771,7 @@ public void setMobileNumberError_doesNotRequestFocusIfCardNumberExpirationCvvPos
 
     @Test
     public void marksCardNumberAsErrorWhenFocusChangesAndCardNumberFailsValidation() {
```

Comment:
An additional test should be added, `marksCardholderNameAsErrorWhenFocusChangesAndCardHolderNameFailsValidation`